src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php

```diff
@@ -19,15 +19,15 @@
 use Adyen\Model\ManagementWebhooks\ObjectSerializer;
 
 /**
- * NotificationDataMessage Class Doc Comment
+ * PaymentMethodRequestRemovedNotificationRequest Class Doc Comment
```

Comment:
do we need to update the managementwebhook parser?

Reply:
I don't think we have one, just webhook models.

Reply:
https://github.com/Adyen/adyen-php-api-library/blob/develop/src/Adyen/Service/ManagementWebhookParser.php#L10 this one?

Reply:
Good catch, file updated!